README.md

```diff
@@ -2,6 +2,17 @@
 
 A CLI tool to simplify your release process!
 
+**Important note**:
+this tool simplify a process, if you don't have one, it is the right time to adopt one!
+
+## The Release Process
+
++ use semver
++ never commit to master, except bump of version
++ squash and commit every PR
```

Comment:
We should support non-squashed PRs. Merge commits are the default for github after all.

Reply:
I think non-squashed PRs are also supported, I'm only suggesting one possible workflow.

I should check if all works as expected with more commits

Reply:
I can confirm that all works also without squash

The commit messages from here are all evaluated as a linked list:
![image](https://user-images.githubusercontent.com/11404065/56609519-99a75880-660d-11e9-9814-91607f2e6bd2.png)

Reply:
So, can we remove these?